### PR TITLE
Test E2E: Modify 'InstallCheByOperatorHub' to use it for update application

### PR DESCRIPTION
**Pull request by artaleks9**
### What does this PR do?
* Add new variable _TS_SELENIUM_UPDATE_INSTALLED_APPLICATION_ to use E2E test for check update installed application using OperatorHub
* Modify 'InstallCheByOperatorHub' to use it for update application

### What issues does this PR fix or reference?

### CODE REVIEW

**Comment by dmytro-ndp** [comment]
What about using more straightforward name 'TS_SELENIUM_UPDATE_CHE_BY_OPERATOR_HUB'?

File: tests/e2e/tests/e2e_install_che_operatorhub/InstallCheByOperatorHub.spec.ts [hunk 1]
```diff
@@ -65,6 +64,10 @@ suite('E2E', async () => {
         });
     });
 
+    if (TestConstants.TS_SELENIUM_UPDATE_INSTALLED_APPLICATION) {
```
Comment (rhopp) [inline tests/e2e/tests/e2e_install_che_operatorhub/InstallCheByOperatorHub.spec.ts]: I haven't had a time to try this, but do I understand it correctly, that this section makes it that no further suites/tests are executed, when `TS_SELENIUM_UPDATE_INSTALLED_APPLICATION=true`?
If yes, this doesn't seem right to me, as there is no validation, that the update was succesfull.
Maybe even more I think updating should have separate spec.ts file to more distinguish these two scenarios (install vs update). And this variable wouldn't be needed at all.

**Review by rhopp — CHANGES_REQUESTED** [review]

File: tests/e2e/tests/e2e_install_che_operatorhub/InstallCheByOperatorHub.spec.ts (hunk 1 above)
Comment (artaleks9) [reply tests/e2e/tests/e2e_install_che_operatorhub/InstallCheByOperatorHub.spec.ts]: @dmytro-ndp I don't mind..

@rhopp Really this change needs to check and automate a private case updating CHE6 to CHE7 and downstream projects using operatorHub which is not supported automate update. So we should do defined actions on web-console OCP. Actually we have a flow: install an application using old operator,    remove subscription old operator and then installed new operator to an existing application. 
But I'm not sure that we need a separate E2E test for this, because it will fully duplicate the part of steps from already exist `InstallCheByOperatorHub.spec.ts`. 
If the `TS_SELENIUM_UPDATE_INSTALLED_APPLICATION=true` the `InstallCheByOperatorHub.spec.ts` performs only part steps needed to subscribe to a new operator and ends.
You're right about verifying success updating, but I think it should be checked using `oc` client in the deployment or something like that in the job which tests this scenario.

Comment (artaleks9) [reply tests/e2e/tests/e2e_install_che_operatorhub/InstallCheByOperatorHub.spec.ts]: @rhopp 
Done.

**Review by rhopp — APPROVED** [review]
